Resources/Prototypes/Nyanotrasen/Entities/Objects/Books/hyperlinks.yml

```diff
@@ -66,8 +71,11 @@
   - type: Sprite
```

Comment:
I can't comment on it from here 🙃

This should be named Cook Book
Description should be The Chemist's guide to "cooking!"